packaging/dashboard.version

```diff
@@ -0,0 +1 @@
+v1.0-alpha.4
```

Comment:
why it is different from this
https://github.com/netdata/netdata/pull/8363/files?file-filters%5B%5D=.am&file-filters%5B%5D=.checksums&file-filters%5B%5D=.in&file-filters%5B%5D=.sh&file-filters%5B%5D=.version#diff-15eb7dcc98aba18503e2174a75a5b21b

Reply:
ok, i see this one is the latest on was released 31 minutes ago, #8363 is not updated